github/repos.go

```diff
@@ -248,6 +249,17 @@ func (d DependabotSecurityUpdates) String() string {
 	return Stringify(d)
 }
 
+// Specifies the state of secret scanning validity checks on a repository.
+//
+// Github API docs: https://docs.github.com/en/enterprise-cloud@latest/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-security-and-analysis-settings-for-your-repository#allowing-validity-checks-for-partner-patterns-in-a-repository
```

Comment:
"Github" => "GitHub" - but if you run the commands in step 4 of CONTRIBUTING.md, then this will most likely get fixed automatically.